### PR TITLE
add deploy instructions to readme [risk: no]

### PR DESCRIPTION
**Pull request by davidangb**
`README.md` update only.

This adds instructions for deploying the TOS API inside the Broad.

### CODE REVIEW

**Review by zarsky-broad — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -25,3 +25,18 @@ To install third-party libraries, first `cd function`, then `npm install`. You w
 1. `cd function` - make sure you're in the right directory. The root of this repostory is NOT the right directory!
 2. `npm install` - get all the third-party dependencies, if you have not done so already. `npm i` is the shorthand.
 3. `npm run lint` - note you need the extra `run` command.
+
+# Broad Internal
+This section applies only to running the workbench-tos codebase within the Broad Institute.
+
+## Deploying
+To deploy the TOS API:
+1. If you are deploying a new version of the codebase, create a tag or release within github for the commit you want to deploy. By convention, we only deploy tagged commits; we do not care about branches of this repo for deploys. If you simply need to deploy/redeploy a pre-existing tag/release, you can skip this step.
```
Comment (andy7i) [inline README.md:34]: Is there a naming convention you'd like others to follow?

File: README.md [hunk 2]
```diff
@@ -25,3 +25,18 @@ To install third-party libraries, first `cd function`, then `npm install`. You w
...
+2. Navigate to the `workbench-tos-manual-deploy` Jenkins job in dev Jenkins, and choose "Build with Parameters". Specify appropriate parameters:
```
Comment (andy7i) [inline README.md:35]: the jenkins job shows this message:
`This item has been changed manually since it was generated by the seed job.`

**Review by andy7i — APPROVED** [review]
Looks reasonable. A couple comments but otherwise 👍

File: README.md (hunk 1 above)
Comment (davidangb) [reply README.md:34]: nope

File: README.md (hunk 2 above)
Comment (davidangb) [reply README.md:35]: yeah, that was me just poking around, it's fine to overwrite

**Pull request merged**